miq_top_output_processor/plot_all

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+#
+# usage: plot_all [-sfpoHUPDlirSE] LOG_FILES_TO_PROCESS
```

Comment:
Kinda want to add some options here to automate the `LOG_FILES_TO_PROCESS` bit better.  The command gets pretty long, and if you transition from a `.partial.gz` to a regular file, and us file globbing, this ends up not working too well.